### PR TITLE
Heap refinement Part1 -- prerequisites

### PR DESCRIPTION
**Pull request by keyonjie**
This is the prerequisites of the Heap refinement PR.

### CODE REVIEW

**Comment by keyonjie** [comment]
@lgirdwood CI all green here.

File: zephyr/wrapper.c [hunk 1]
```diff
@@ -72,18 +72,33 @@ __section(".heap_mem") static uint8_t __aligned(64) heapmem[HEAPMEM_SIZE];
 
 #else
 
-#define HEAPMEM_SHARED_SIZE	(HEAP_SYSTEM_SIZE + HEAP_RUNTIME_SIZE + \
-				HEAP_RUNTIME_SHARED_SIZE + HEAP_SYSTEM_SHARED_SIZE)
+/* TODO: heap size should be set based on linker output */
```
Comment (lgirdwood) [inline zephyr/wrapper.c:75]: Can we also comment on where we get these HEAP_SIZE numbers from (assuming default values used by Zephyr builds today based on our conversations).

**Review by lgirdwood — COMMENTED** [review]
Just need an extra comment.

File: zephyr/wrapper.c (hunk 1 above)
Comment (keyonjie) [reply zephyr/wrapper.c:75]: They come from the XTOS linker scripts (e.g. src/platform/apollolake/apollolake.x.in for APL):
(_bss_end - _system_heap_start)

Sure, let me add this.

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**